.gitlab-ci.yml

```diff
@@ -0,0 +1,41 @@
+test-installer:
+  stage: test
+  image: centos:8
+  script:
+    - |
+      set +e
+      set -x
+
+      # usage
+      stdout=$(bash crucible-install.sh --help)
```

Comment:
Small nit...do we have to run it as `bash crucible-install.sh` as opposed to `./crucible-install.sh`?  I suspect most users would use the `./crucible-install.sh` form so that seems like what we should test.

Reply:
Agreed. chmod +x on it now :)